ncm-metaconfig/src/main/pan/components/metaconfig/config-common.pan

```diff
@@ -1,17 +0,0 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
-# ${build-info}
-
-unique template components/${project.artifactId}/config-common;
-
-include { 'components/${project.artifactId}/schema' };
-
-# Set prefix to root of component configuration.
-prefix '/software/components/${project.artifactId}';
-
-#'version' = '${project.version}';
-#'package' = 'NCM::Component';
```

Comment:
I was going to comment that the version attribute had been dropped which might prevent the component being run after a new version was installed.  However now I remember that following a schema change to /software/packages, all components were dispatched, so does ncm-ncd already include functionality to register  the /software/packages/ncm-$component/ subtree for changes? In which case the version attribute should  be deprecated and removed from other PRs such as #601 ?